broker-cli/cmd/flags/brokerurl.go

```diff
@@ -29,6 +38,22 @@ func ConstructBrokerURL(host, project, broker string) string {
 	return fmt.Sprintf("%s/v1beta1/projects/%s/brokers/%s", host, project, broker)
 }
 
+func (flags *BrokerURLConstructor) validateServer() error {
+	if strings.HasPrefix(flags.Server, flags.Host) == false {
+		return fmt.Errorf("broker server URL %q should always begin with service end point %q", flags.Server, flags.Host)
+	}
+
+	parts := strings.Split(flags.Server, "/")
+	len := len(parts)
+
+	if parts[len-2] == brokersKey && parts[len-4] == projectsKey && parts[len-5] == betaKey {
```

Comment:
nit: invert check so error flow is indented

Reply:
done.